cloud/store/schema.go

```diff
@@ -32,19 +32,21 @@ var (
 )
 
 type schema struct {
+	sync.RWMutex
+
 	nodeID      string
 	shardReader shardReader
-	sync.RWMutex
```

Comment:
The conversion of having the mutex above the data structure makes it clear that the mutex is specifically protecting this DS